es/account/memory.js

```diff
@@ -59,7 +63,7 @@ function validateKeyPair (keyPair) {
 }
 
 /**
- * In-memory `Account` factory
+ * In-memory `Account` factory. The resulting account `stamp` is used in various functions for setting up and configuring the SDK.
```

Comment:
I would say that account just provides and address and a signing method.

Reply:
Logging the outcome resulted in something very odd for me, and the address was empty also, can you check please?

Reply:
```js
const { Crypto, MemoryAccount } = require('./dist/aepp-sdk');

(async () => {
  const keypair = Crypto.generateKeyPair();
  console.log(keypair);
  const memoryAccount = MemoryAccount({ keypair });
  console.log(memoryAccount);
  console.log(await memoryAccount.address());
  console.log(await memoryAccount.sign('test'));
})();
```
Outputs:
```
$ node test.js
{
  publicKey: 'ak_2aBDpnmDY6k8nTVQQg2QheXn2rZPsjecpwydRqkhzKD9HtKDnB',
  secretKey: '8da5c2aa66f1eff003502f526070a0efcaa66bcd25ef1a3a45535f6b7bb24a8acf06dab6590d534589d61d1357e532cdd48926da366e21ea2d8afdb379776d56'
}
{ isGa: false }
ak_2aBDpnmDY6k8nTVQQg2QheXn2rZPsjecpwydRqkhzKD9HtKDnB
Uint8Array(64) [
  101, 144,  50, 210, 154, 194, 194, 133, 226, 192, 177,
  181, 164, 124, 214, 163, 217,  33, 134, 160,  13,   6,
    0,  58, 169,  80,  97, 121,  23, 198, 254, 195, 118,
   96, 255,  43, 156,  78, 101, 219, 120,  59, 223,   4,
  209, 111,  36, 120, 113,   1, 122,  78, 172, 244, 195,
  107, 208,  23,  72, 144,  11, 245, 131,   7
]
```
it looks ok to me

Reply:
ah indeed, I forgot they were async. that gives a good example code code btw, thanks!